src/assets/css/main.css

```diff
@@ -434,3 +434,6 @@ br.big {
 em-emoji-picker {
   width: 100%;
 }
+.overflow-hidden {
```

Comment:
I think there's already a class for this in bootstrap.

Reply:
You're absolutely right, I somehow missed that. Removed the CSS rule.